docs/modules/ROOT/pages/how-tos/metadata/index.adoc

```diff
@@ -1 +1,56 @@
+
+Don't take our word on the transparency -- you can download and inspect these artifacts yourself! {ProductName} generates and pushes build metadata as OCI artifacts to the container registry alongside your software's build using the link:https://github.com/oras-project/artifacts-spec/blob/main/manifest-referrers-api.md[referrers API].
+
+== Discovering the associated metadata
+
+.Prerequisites
+
+* Install the link:https://docs.sigstore.dev/cosign/installation/[Cosign] CLI tool.
+
+* Install the link:https://stedolan.github.io/jq/download/[jq] CLI tool.
+
+* xref:/getting-started/cli.adoc[Login] to {ProductName}.
+
+.Procedure
+
+In the CLI, complete the following steps to discover the associated metadata a component:
+
+. Find the image for a xref:/how-tos/creating.adoc#finding-the-built-image[recently completed build pipeline] and export the value to the `IMAGE` environment variable
++
+Example:
++
+[source]
+--
+IMAGE=quay.io/redhat-user-workloads/workspace-tenant/application/component@sha256:<output omitted>
+--
+
++
+. Print a high-level overview of the artifact's metadata that {ProductName} has created for a component: 
+
++
+[source]
+--
+cosign tree $IMAGE
+--
++
+Example output:
++
+[source]
+--
+📦 Supply Chain Security Related artifacts for an image: quay.io/redhat-user-workload/workspace-tenant/application/component@sha256:<sha-digest>
+└── 📦 SBOMs for an image tag: quay.io/redhat-user-workload/workspace-tenant/application/component:sha256-<sha-digest>.sbom
+   └── 🍒 sha256:<sbom-sha-digest>
+└── 💾 Attestations for an image tag: quay.io/redhat-user-workload/workspace-tenant/application/component:sha256-<sha-digest>.att
+   └── 🍒 sha256:<attestation-sha-digest>
+└── 🔐 Signatures for an image tag: quay.io/redhat-user-workload/workspace-tenant/application/component:sha256-<sha-digest>.sig
+   └── 🍒 sha256:<signature-sha-digest>
+--
+
+The `cosign tree` command will show all metadata associated to the artifact via the referrers API. The SBOM artifact contains the Software Bill of MaterialsWe have provided further procedures for you to view and inspect the xref:/how-tos/metadata/attestations.adoc[attestations] and xref:/how-tos/metadata/sboms.adoc[SBOMs] by {ProductName}.
```

Comment:
"...Software Bill of Materials. We have provided..."

Currently missing a period and space there.

Reply:
```suggestion
The `cosign tree` command will show all metadata associated to the artifact via the referrers API. The SBOM artifact contains the Software Bill of Materials. We have provided further procedures for you to view and inspect the xref:/how-tos/metadata/attestations.adoc[attestations] and xref:/how-tos/metadata/sboms.adoc[SBOMs] by {ProductName}.
```